tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>This comment can be omitted in my opinion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>